ci/python_test.sh

```diff
@@ -5,7 +5,7 @@ set -euxo pipefail
 VENV=env
 
 rm -rf "$VENV"
-python -mvenv "$VENV"
+virtualenv -p $(which python) "$VENV"
```

Comment:
newer python installs don't necessarily have virtualenv, but py3 is guaranteed to have -mvenv

Reply:
I have reverted my changes. I'll create a separate issue if we want to add python2 CI.